Description: Fix FTBFS on GNU/kFreeBSD due to an undefined reference to __bswap32.
Author: Petr Salinger <Petr.Salinger@seznam.cz>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=566776

--- a/src/libical/icaltz-util.c
+++ b/src/libical/icaltz-util.c
@@ -31,6 +31,9 @@
 # ifdef HAVE_BYTESWAP_H
 #  include <byteswap.h>
 # endif
+# ifdef HAVE_ENDIAN_H
+#  include <endian.h>
+# else
 # ifdef HAVE_SYS_ENDIAN_H
 #  include <sys/endian.h>
 #  ifdef bswap32
@@ -39,8 +42,6 @@
 #   define bswap_32 swap32
 #  endif
 # endif
-# ifdef HAVE_ENDIAN_H
-#  include <endian.h>
 # endif 
 #endif
 
